content/_data/changelogs/weekly.yml

```diff
@@ -14157,6 +14157,169 @@
+        authors:
+          - NotMyFault
+        pr_title: Utilize new tables on fingerprints and expired tokens view
+        message: |-
+          Use modern table design for "Recorder Fingerprints" and "Legacy API Tokens" tables.
+      - type: rfe
+        category: rfe
+        pull: 6259
+        authors:
+          - janfaracik
+        pr_title: Polish forms components and split out CSS files
+        message: |-
+          Minor form and spacing changes
+      - type: rfe
+        category: rfe
+        pull: 6248
+        authors:
+          - janfaracik
+        pr_title: Update table styling and spacing
+        message: |-
+          Update table styling and spacing with small table UI tweaks.
+      - type: rfe
+        category: rfe
+        pull: 6228
+        authors:
+          - basil
+        pr_title: Notify `systemd(1)` about start-up completion and other service
+          status changes
+        message: |-
+          Startup completion notification is available with <code>systemd(1)</code>.
+      - type: rfe
+        category: rfe
+        pull: 6241
+        authors:
+          - daeho-ro
+        pr_title: Update the internationalization of the build icon description for
+          Korean
+        message: |-
+          Update the Korean internationalization for build icon descriptions.
+      - type: rfe
+        category: rfe
+        pull: 6237
+        authors:
+          - basil
+        pr_title: Extend startup notification timeout as each initialization milestone
+          is attained
+        message: |-
+          Extend startup notification timeout as each initialization milestone is attained.
+      - type: bug
+        category: regression
+        pull: 6261
+        authors:
+          - dependabot[bot]
+        pr_title: "[JENKINS-67000] Bump remoting from 4.11.2 to 4.12"
+        message: |-
+          Update remoting from 4.11.2 to [4.12](https://github.com/jenkinsci/remoting/releases/tag/remoting-4.12) to allow Java web start agents to connect (regression in 2.318).
+      - type: bug
+        category: bug
+        pull: 6264
+        issue: 67753
+        authors:
+          - NotMyFault
+        pr_title: '[JENKINS-67753] Overwrite "grey" balls with modern build status'
+        message: |-
+          Overwrite grey balls icon with the modern "nobuilt" status.
+      - type: bug
+        category: bug
+        pull: 6245
+        authors:
+          - NotMyFault
+        pr_title: "[JENKINS-67722] Fix `/configure` link on custom log recorders"
+        message: |-
+          The log recorder configure button links to the associated log recorder (regression in 2.322).
+      - type: bug
+        category: bug
+        pull: 6233
+        authors:
+          - NotMyFault
+        pr_title: "[JENKINS-67609] Deduplicate help button question mark"
+        message: |-
+          Set the question mark on the new help button only once so that it is not shown twice when using different themes.
```

Comment:
```suggestion
          Render the question mark on the new help button only once, even while using different themes.
```

```suggestion
          Render the question mark on the new help button only once so that it is not shown twice while using different themes.
```

Maybe?

Reply:
Updated in https://github.com/jenkinsci/jenkins/pull/6233